.github/workflows/pr.yaml

```diff
@@ -32,5 +32,11 @@ jobs:
           distribution: adopt
           java-version: 11
 
+      - name: Build Windows Collector image
```

Comment:
Do we need to build this for every PR? Or we can just pull the image built by CI job?

Reply:
I'll change the CI job 👍